### DISCUSSION TITLE
Highlight markdown in comments.

### DISCUSSION THREAD

**Discussion by TomPlanche** [zed-industries/zed#7620] [Feature Requests]
When I use [RustRover](https://www.jetbrains.com/rust/), the comments are highlighted.
This is something not important but I really miss it.

<img width="834" alt="Capture d’écran 2024-02-09 à 16 23 03" src="https://github.com/zed-industries/zed/assets/58936594/eb5db685-65c6-480b-8b30-b2b0c18592ea">

And I can 'toggle' it and it looks like this:

<img width="633" alt="Capture d’écran 2024-02-09 à 16 29 05" src="https://github.com/zed-industries/zed/assets/58936594/b2142dae-5a17-4f17-9655-e6ad59d0612f">

Something like this should be cool :).

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
